packages/wallet/ui/src/contexts/Provider.jsx

```diff
@@ -148,11 +154,36 @@ const Provider = ({ children }) => {
   );
   const [keplrConnection, setKeplrConnection] = useState(null);
 
+  /**
+   * NOTE: relies on ambient window.fetch, window.keplr, Random.getBytes
```

Comment:
`assert`s for fetch and keplr make sense, but `Missing Random.getBytes` would be a static error.
`Random.getBytes` is still noteworthy because it is ambient authority, contrary to [OCap Discipline](https://github.com/Agoric/agoric-sdk/wiki/OCap-Discipline).